native/src/widget/scrollable.rs

```diff
@@ -435,18 +509,18 @@ pub fn update<Message>(
         return event::Status::Captured;
     }
 
-    if is_mouse_over {
+    if mouse_over_scrollable {
         match event {
             Event::Mouse(mouse::Event::WheelScrolled { delta }) => {
-                match delta {
-                    mouse::ScrollDelta::Lines { y, .. } => {
-                        // TODO: Configurable speed (?)
-                        state.scroll(y * 60.0, bounds, content_bounds);
-                    }
-                    mouse::ScrollDelta::Pixels { y, .. } => {
-                        state.scroll(y, bounds, content_bounds);
+                let delta = match delta {
+                    mouse::ScrollDelta::Lines { x, y } => {
+                        // TODO: Configurable speed/friction (?)
+                        Vector::new(x * 60.0, y * 60.0)
```

Comment:
Yes! `Vector` makes sense here since this represents a `delta` (i.e. a movement to be applied).